pkg/shell/terraform.go

```diff
@@ -276,15 +326,24 @@ func applyOrDestroy(tf *tfexec.Terraform, b ApplyBehavior, destroy bool) error {
 		return nil
 	}
 
-	if err := applyPlanConsoleOutput(tf, f.Name()); err != nil {
-		return err
+	switch of {
+	case JsonOutput:
+		if err := applyPlanJsonOutput(tf, f.Name()); err != nil {
+			return err
+		}
+	case TextOutput: // Text output to the console is also the default choice
+		if err := applyPlanConsoleOutput(tf, f.Name()); err != nil {
+			return err
+		}
+	default:
+		panic(fmt.Sprintf("Unsupported output format requested. Supported output formats: %s", supportedOutputFormats))
```

Comment:
Don't use `Sprintf` here. 
a) yet another change to derail on panic-path;
b) no need for nice output;
c) constant defined somewhere else hurts readability